src/commands/TrackerCommand.ts

```diff
@@ -0,0 +1,580 @@
+						sub
+							.setName("remove")
+							.setDescription("remove a game from your blacklist")
+							.addStringOption((opt) =>
+								opt
+									.setName("game")
+									.setDescription("Enter game or dont enter anything to enable your logging")
+									.setAutocomplete(true)
+									.setRequired(true)
+							)
+					)
+					.addSubcommand((sub) => sub.setName("show").setDescription("See what is on your blacklist"))
+			)
+			.addSubcommandGroup((group) =>
+				group
+					.setName("statistics")
+					.setDescription("Show statistics")
+					.addSubcommand((sub) =>
+						sub
+							.setName("mystats")
+							.setDescription("Show your activity statistics")
+							.addStringOption((opt) =>
+								opt
+									.setName("game")
+									.setDescription("Show statistics for a specific game")
+									.setAutocomplete(true)
+							)
+					)
+					.addSubcommand((sub) =>
+						sub
+							.setName("gamestats")
+							.setDescription("Show statistics for a specific game across all users")
+							.addStringOption((opt) =>
+								opt
+									.setName("game")
+									.setDescription("Show statistics for a specific game")
+									.setAutocomplete(true)
+									.setRequired(true)
+							)
+					)
+					.addSubcommand((sub) =>
+						sub.setName("allstats").setDescription("Show statistics about all games across all users")
+					)
+			)
+			.addSubcommandGroup((group) =>
+				group
+					.setName("admin")
+					.setDescription("admin only commands")
+					.addSubcommand((sub) =>
+						sub
+							.setName("reset")
+							.setDescription("Reset every log and blacklist entry")
+							.addBooleanOption((opt) =>
+								opt.setName("sure").setDescription("Are you really sure?").setRequired(true)
+							)
+							.addStringOption((opt) =>
+								opt
+									.setName("really")
+									.setDescription("Are you really sure you want to delete every entry?")
+									.addChoices(
+										{ name: "No. I dont want to delete every log and blacklist entry!", value: "no" },
+										{ name: "Yes I am sure. I want to delete every log and blacklist entry!", value: "yes" }
+									)
+									.setRequired(true)
+							)
+					)
+					.addSubcommand((sub) =>
+						sub
+							.setName("blacklistgame")
+							.setDescription("Blacklist a game for all users.")
+							.addStringOption((opt) =>
+								opt
+									.setName("game")
+									.setDescription("The game which should get blacklisted globaly")
+									.setRequired(true)
+							)
+					)
+					.addSubcommand((sub) =>
+						sub
+							.setName("whitelistgame")
+							.setDescription("Remove a game from the global blacklist")
+							.addStringOption((opt) =>
+								opt
+									.setName("game")
+									.setDescription("The game which should get removed from the global blacklist")
+									.setRequired(true)
+									.setAutocomplete(true)
+							)
+					)
+					.addSubcommand((sub) =>
+						sub
+							.setName("look")
```

Comment:
shouldnt there also be a way to view the global blacklist

Reply:
Autocomplete shows the content of the blacklist, but yes